app/models/solr_document.rb

```diff
@@ -46,6 +26,24 @@ class SolrDocument
   # Use field names from Dublin Core
   use_extension(Blacklight::Document::DublinCore)
 
+  # Blacklight OAI metadata mapping
+  field_semantics.merge!(
+    title:       "title_tesim",
+    description: "description_tesim",
+    langauge: "language_label_ssm",
+    subject: "lc_subject_label_tesim",
+    contributor: "all_contributors_label_sim",
+    coverage: "location_label_tesim",
+    creator: "creator_tesim",
+    date: "date_si",
+    format: "form_of_work_label_tesim",
+    identifier: ["identifier_ssm"],
+    publisher: "publisher_tesim",
+    relation: %w[collection_label_ssim square_thumbnail_url_ssm],
+    rights: "license_tesim",
+    type: "human_readable_type_tesim"
```

Comment:
I believe it is the first, like "Audio Recording", "Image" "Map Set"

Reply:
The Fedora object type doesn't always correspond to the actual type of the object (e.g., we use the Image type when ingesting the newspapers). It would be better to use the type of resource field for this (the one with values from http://id.loc.gov/vocabulary/resourceTypes.html)

Reply:
I think the correct field to use is "work_type_label_tesim"